acvm/src/pwg.rs

```diff
@@ -36,7 +40,11 @@ pub enum PartialWitnessGeneratorStatus {
     ///
     /// The caller must resolve these opcodes externally and insert the results into the intermediate witness.
     /// Once this is done, the `PartialWitnessGenerator` can be restarted to solve the remaining opcodes.
-    RequiresOracleData { required_oracle_data: Vec<OracleData>, unsolved_opcodes: Vec<Opcode> },
+    RequiresOracleData {
+        required_oracle_data: Vec<OracleData>,
+        unsolved_opcodes: Vec<Opcode>,
+        unresolved_brilligs: Vec<UnresolvedBrillig>,
```

Comment:
Can we add a comment on what the caller is expected to do with this field? I get a set of bytecode and something called a "ForeignCallWaitInfo" but what do I do with it?

I presume this is wanting an oracle response but where do I write it?

Reply:
Added an issue for this: https://github.com/noir-lang/acvm/issues/309